src/commands/index.js

```diff
@@ -1,48 +1,6 @@
 // @flow
 module.exports = {
   ...require('./add'),
```

Comment:
ignore this file change, it is just to that we can test stuff, everything will come back in once all commands are updated.